mkosi/config.py

```diff
@@ -4562,6 +4569,7 @@ def summary(config: Config) -> str:
                          Passphrase: {none_to_none(config.passphrase)}
                            Checksum: {yes_no(config.checksum)}
                                Sign: {yes_no(config.sign)}
+                       OpenPGP Tool: ({config.openpgp_tool or "gpg"})
```

Comment:
Just so we don't forget: This also needs an entry in the man page.

Reply:
Entry added. See if looks okay.